### DISCUSSION TITLE
Twitter api v2?

### DISCUSSION THREAD

**Discussion by vanaf1979** [nextauthjs/next-auth#3713] [Help] [answered]
Is it possible to get the accessOAuthToken and accessOAuthSecret or is the Twitter Api v2 not supported yet?

If it is is it also possible to set scopes?

Thanks

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/providers/twitter#oauth-2

please refer to the rest of the documentation. all those questions are answered
